README.md

```diff
@@ -1,64 +1,92 @@
-# **OSSF Endusers**
+# OSSF End Users
 
-[Brief description of the initiative]
+Anyone is welcome to join our open discussions related to the group's mission and charter.
```

Comment:
Why mention the mission and charter here? It feels like it could just say "Anyone is welcome to join our open discussions".